GenderPayGap.WebUI/Services/OrganisationService.cs

```diff
@@ -134,6 +125,49 @@ public Organisation ImportOrganisationFromCompaniesHouse(string companyNumber, U
             return organisation;
         }
 
+        private Organisation SetDeletedRetiredOrganisationToActive(Organisation existingOrganisation, User requestingUser)
+        {
+            var oldStatus = existingOrganisation.Status;
+
+            existingOrganisation.SetStatus(OrganisationStatuses.Active, requestingUser.UserId, "Automatic reactivation");

```

Comment:
What if the company data (e.g., the address) has been updated in the meantime? Should we do that as well?

Reply:
There is a job FetchCompaniesHouseDataJob that handles this kind of thing (not sure on the schedule) that does it for all where they aren't OptedOutFromCompaniesHouseUpdate and have a CompanyNumber. There's no filter for status on it unless we're setting OptedOutFromCompaniesHouseUpdate somewhere I haven't seen.

Since our existingOrganisation has been searched by CompanyNumber, it should have been picked up by this job, so we shouldn't need to.

Reply:
Great then 👍 

You can see details about the background jobs in `BackgroundJobsApi.cs`. It seems that FetchCompaniesHouseDataJob runs every 5 minutes.